netmaster/master/netmaster.go

```diff
@@ -36,6 +36,13 @@ const (
 	defaultSkyDNSImage  = "skynetservices/skydns:latest"
 )
 
+// Run Time config of netmaster
+type nmRunTimeConf struct {
+	clusterMode string
+}
+
+var masterRTCfg nmRunTimeConf
```

Comment:
@jainvipin: my thinking is:
(1) Plugin modes (like docker/kubernetes/mesos plugins) that need to be specified before netplugin/netmaster initializes should be command line arguments.
(2) Cluster modes/Fabric modes/control plane modes should be an Intent that can change at runtime. We need little more infra in netplugin/netmaster for this to happen. Expect to work on this in next few PRs.